core/impl/src/main/java/com/blazebit/persistence/impl/JoinManager.java

```diff
@@ -385,12 +390,15 @@ public Object getValue(Object target) {
                 parameterNames[j][0] = prefix + '_' + attributeName + '_' + j;
             }
         } else if (identifiableReference) {
-            Attribute<?, ?> attribute = attributeSet.iterator().next();
-            String attributeName = attribute.getName();
-            attributes[0] = attributeName;
-            pathExpressions[0] = com.blazebit.reflection.ExpressionUtils.getExpression(clazz, attributeName);
-            for (int j = 0; j < valueCount; j++) {
-                parameterNames[j][0] = prefix + '_' + attributeName + '_' + j;
+            int i = 0;
```

Comment:
Functionally the same as else clause, can be removed.

Reply:
I agree with that